charts/n8n/values.yaml

```diff
@@ -630,6 +644,13 @@ webhook:
   tolerations: []
   affinity: {}
 
+  # extraContainers is a list of sidecar containers. Specified as a YAML list.
+  extraContainers: null
+  # extraContainers:
+  #   - name: my-sidecar
+  #     image: busybox:latest
+  #     command: ["sh", "-c", "echo hello"]
```

Comment:
A sidecar is expected to stay running. This example doesn't do that.
Changing this to something like the example below will keep it running constantly. 

```command: ["sh", "-c", "while true; do echo hello && sleep 60; done"]```